src/main.ts

```diff
@@ -1,33 +1,48 @@
-import {setFailed} from '@actions/core';
+import {setFailed, endGroup, info, startGroup} from '@actions/core';
 import {context} from '@actions/github';
+import {promises as fs} from 'fs';
 import process from 'process';
 import {headSHA} from './git';
 import {getInputs} from './inputs';
-import * as proxy from './proxy';
+import {execSync} from 'child_process';
 import * as sonarScanner from './sonar-scanner';
 import * as maven from './maven';
-import * as state from './state';
 import * as args from './args';
 
 async function run(): Promise<void> {
   try {
-    if (state.isPost) {
-      await post();
-      return;
-    }
-    state.setIsPost();
-
     const inputs = await getInputs();
-    let sonarHost =
-      'http://sonarqube-default-sonarqube-0.sonarqube-default-sonarqube.default.svc.cluster.local:9000';
+    startGroup('Selecting sonar host');
```

Comment:
what's this?

Reply:
Comment groups for the sonar job
![image](https://github.com/Tradeshift/actions-sonarqube/assets/59831933/1343bac2-6b11-42b6-8468-8e8f0fb12649)